ruma-events/tests/room_message.rs

```diff
@@ -26,11 +26,15 @@ use serde_json::{from_value as from_json_value, json, to_value as to_json_value}
 #[test]
 fn serialization() {
     let ev = MessageEvent {
-        content: MessageEventContent::Audio(AudioMessageEventContent {
-            body: "test".into(),
-            info: None,
-            url: Some("http://example.com/audio.mp3".into()),
-            file: None,
+        content: assign!(MessageEventContent::new(
+            MessageType::Audio(AudioMessageEventContent {
+                body: "test".into(),
+                info: None,
+                url: Some("http://example.com/audio.mp3".into()),
+                file: None,
+            }),
+        ), {
+            relates_to: None,
```

Comment:
This writes `None` to a field that's already `None`, right? I think this entire `assign!` is not needed.